app.js

```diff
@@ -37,3 +38,147 @@ routes.init(app);
 app.use(errors.handle);
 
 module.exports = app;
+
```

Comment:
Delete all comments.
There shouldn't be any comments unless they're really really necessary.